python/lsst/ts/watcher/rules/mtm2_total_force_moment.py

```diff
@@ -0,0 +1,204 @@
+                    type: number
+                    default: 17600.0
+                fz:
+                    description: >-
+                        Limit of the total force in z-direction (N).
+                    type: number
+                    default: 17600.0
+                mx:
+                    description: >-
+                        Limit of the total moment in x-direction (N * m).
+                    type: number
+                    default: 1500.0
+                my:
+                    description: >-
+                        Limit of the total moment in y-direction (N * m).
+                    type: number
+                    default: 40.0
+                mz:
+                    description: >-
+                        Limit of the total moment in z-direction (N * m).
+                    type: number
+                    default: 300.0
+                poll_interval:
+                    description: Time delay between polling updates (second).
+                    type: number
+                    default: 1.0
+
+            required:
+            - fx
+            - fy
+            - fz
+            - mx
+            - my
+            - mz
+            - poll_interval
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    def setup(self, model) -> None:
+
+        self._remote = model.remotes[("MTM2", 0)]
+
+    def compute_alarm_severity(self) -> AlarmSeverityReasonType:
+        """Compute and set alarm severity and reason.
+
+        Returns
+        -------
+        None, if no change or unknown, or a tuple of two values:
+
+        severity: `lsst.ts.xml.enums.Watcher.AlarmSeverity`
+            The new alarm severity.
+        reason : `str`
+            Detailed reason for the severity, e.g. a string describing
+            what value is out of range, and what the range is.
+            If ``severity`` is ``NONE`` then this value is ignored (but still
+            required) and the old reason is retained until the alarm is reset
+            to ``nominal`` state.
+
+        Notes
+        -----
+        You may return `NoneNoReason` if the alarm state is ``NONE``.
+        """
+
+        assert self._remote is not None
+
+        list_forces = list()
+        if self._remote.tel_netForcesTotal.has_data:
+            list_forces = self._check_out_of_range(
+                self._remote.tel_netForcesTotal.get(), ["fx", "fy", "fz"]
+            )
+
+        list_moments = list()
+        if self._remote.tel_netMomentsTotal.has_data:
+            list_moments = self._check_out_of_range(
+                self._remote.tel_netMomentsTotal.get(), ["mx", "my", "mz"]
+            )
+
+        if (len(list_forces) != 0) and (len(list_moments) != 0):
+            return (
+                AlarmSeverity.SERIOUS,
+                f"Force ({list_forces}) and moment ({list_moments}) out of normal range.",
```

Comment:
See my other comment about making reason a static string.

Reply:
Fixed in the commit: [e5e703b](https://github.com/lsst-ts/ts_watcher/pull/83/commits/e5e703b275b024b0fd34185ef4cae12882298a75). Thanks!